scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -135,6 +137,7 @@ AIDriveStrategyUnloadCombine.myStates = {
     BACKING_UP_FOR_REVERSING_COMBINE = { vehicle = nil }, -- reversing as long as the combine is reversing
     MOVING_AWAY_FROM_BLOCKING_VEHICLE = { vehicle = nil }, -- reversing until we have enough space between us and the combine
     WAITING_FOR_MANEUVERING_COMBINE = {},
+    DRIVING_TO_INVERTED_GOAL_POSITION_MARKER = {} --- Drives there before giving control to AD or giants unloader
```

Comment:
Can we call this something more descriptive, like DRIVING_BACK_TO_START_POSITION(_WHEN_FULL?) ? 

Inverted position marker is an implementation detail, I understand it needs to be reversed and offset to avoid collision, but this should be explained in a comment, the point here is that we are driving back to the start position and probably give control back.